src/utils/forms.js

```diff
@@ -56,7 +55,6 @@ ValidationError.prototype.name = 'ValidationError';
  */
 export function createForm({ validateFields, ...formConfig }) {
   return FormComponent => {
-    @autobind
     class WrappedForm extends React.Component {
```

Comment:
The reason I removed autobind was because this was causing a very cryptic warning, which I unfortunately seem to have lost because `yarn run build` clears the screen. But basically, it was complaining that it can't decorator a duplicate named component. Switching to class method fields solves the problem.